taker-frontend/src/components/LiquidationPrice.ts

```diff
@@ -0,0 +1,16 @@
+function quantoLiquidationPrice(leverage: number, quantity: number, price: number) {
+    return (price * (1 / leverage));
+}
+
+// returns the liquidation price for quanto contracts e.g. ETHUSD
+export function liquidationPriceQuantoContracts(leverage: number, quantity: number, price: number, isLong: boolean) {
+    if (isLong) {
+        const liquidationPrice = price - quantoLiquidationPrice(leverage, quantity, price);
+        if (liquidationPrice <= 0) {
+            return 1;
```

Comment:
🔧 I think this should be zero. It's only not allowed to be zero in the daemon because of how we model `Price`.